Scheduler/v7/maintel_fbs_sit_block_t248.yaml

```diff
@@ -0,0 +1,55 @@
+maintel:
+  driver_type: feature_scheduler
+  mode: ADVANCE
+  startup_type: COLD
+  predicted_scheduler_window: 0.5
+  startup_database: ' '
```

Comment:
In order for the scheduler to recover from fault correctly, we need to make sure it replays the targets that were already observed. For  that you need to pass the path to the database so this should be:

```
startup_database: /home/saluser/rubin_sim_data/fbs_observation_database_maintel.sql
```

e.g. the same value used later for `observation_database_name`